IDrive Cloud S3(us-east-1).cyberduckprofile

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>idrivecloud</string>
+        <key>Description</key>
+        <string>IDrive Cloud Storage (us-east-1)</string>
+        <key>Default Hostname</key>
+        <string>s3.us-east-1.idrivecloud.io</string>
```

Comment:
From what I know an endpoint specific to an access key created is to be used. See https://www.idrive.com/e2/cyberduck